### DISCUSSION TITLE
JDP support

### DISCUSSION THREAD

**Discussion by bystrzak** [adoptium/adoptium-support#608] [Q&A]
I am trying to get [JDP](https://docs.oracle.com/javase/10/management/java-discovery-protocol.htm#JSMGM-GUID-4FE44ECD-E1A7-42CE-9AD8-A2E582C55C43) working on Temurin. I tried multiple JDK releases and multiple sets of parameters with no luck. I am confident that the network is not an issue as I sent some multicast traffic across the machines I was testing on.
Is JDP even supported in Temurin?

**Comment by karianna** [comment]
@bystrzak Have you tried other OpenJDK providers such as Azul's Zulu or Amazon's Corretto?  I'm wondering if this is an OracleJDK only feature these days.

**Reply by bystrzak** [reply]
No, I have not so I do not know whether it works with the others or not

**Reply by karianna** [reply]
Would you mind giving one of them a go quickly?

**Reply by vijaya-lakshmi-venkatraman** [reply]
Following.
